src/megaapi_impl.cpp

```diff
@@ -19955,6 +20117,24 @@ void MegaApiImpl::sendPendingRequests()
             }
             break;
         }
+        case MegaRequest::TYPE_GET_REGISTERED_CONTACTS:
+        {
+            const auto contacts = request->getMegaStringMap();
+            if (contacts)
+            {
+                client->reqs.add(new CommandGetRegisteredContacts{client, *contacts});
+            }
+            else
+            {
+                assert(false && "contacts must be valid");
```

Comment:
we should also set                 e = API_EARGS;  so that non-debug builds behave as expected (listeners always complete, and with an appropriate error code)

Reply:
Will do